python/wpt/run.py

```diff
@@ -150,7 +155,8 @@ def run_tests(default_binary_path: str, **kwargs):
         for result in unexpected_results:
             result.flaky = result.path not in stable_tests
 
-        all_filtered = filter_intermittents(unexpected_results,
+        all_filtered = filter_intermittents(github_context,
+                                            unexpected_results,
```

Comment:
Another option, which might be a bit less fragile would be to push the environment variable back here, after tests are run.